src/index.html

```diff
@@ -2,11 +2,410 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="../styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Henny+Penny&family=Lexend+Deca:wght@400;500;600;700&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+
+    <header class="header-menu">
+      <a href="#home" class="header-menu__image">
+        <img
+        src="../src/images/logo.png"
+        alt="moyo logo"
+        class="logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__list-item">
+            <a
+              href="#"
+              class="nav__list-tag is-active"
+               >
```

Comment:
```suggestion
            <a
              href="#"
              class="nav__list-tag is-active"
            > 
```
The closing tag should be just under the opening.
Please fix all cases